frontend/src/components/common/Calendar/WeeklyEventItemPopUp.tsx

```diff
@@ -118,7 +118,7 @@ const WeeklyEventItemPopUp = ({
                       textStyle={isMobile ? "mobileSmall" : "desktopSmall"}
                       py="0.5rem"
                     >
-                      Next Dropoff:{" "}
+                      Next dropoff:{" "}

```

Comment:
@Shehryar21 side note idk if this code is still being used but this should be removed right

Reply:
yeah this is not used, will clean up sometime later

Reply:
we're not using this component anymore so removing it or not shouldn't reflect anything for the user ^^